### DISCUSSION TITLE
Can I debug Jupyter Notebook on remote JupyterHUB server?

### DISCUSSION THREAD

**Discussion by japineda3** [microsoft/vscode-jupyter#15052] [Questions and Answers]
Hi,
I have Vscode on local and I connect via Jupyter Extension with a remote  JupyterHUB server. I want to debug the Jupyter Notebook file on vscode runs on the remote JupyterHUB . Is it posible?
Thanks on advance

**Comment by DonJayamanne** [comment]
Yes you can.

**Comment by DonJayamanne** [comment]
If it does not work, please do file an issue.

**Comment by japineda3** [comment]
Hi,
   Could you explain as do it, are they documentation about this?
Thanks 

Enviado desde Yahoo Mail con Android 
 
  El lun., ene. 22, 2024 a 23:24, Don ***@***.***> escribió:   


If it does not work, please do file an issue.
